tests/Tooltip.test.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-
+import mountWithTheme from './utils/mountWithTheme';
 import Tooltip from '../src/Tooltip';
 
 describe('<Tooltip />', () => {
   it('properly renders a tooltip', () => {
-    const tree = shallow(
+    const tree = mountWithTheme(
```

Comment:
I couldn't find anything that says `jest-emotion` (https://github.com/emotion-js/emotion/tree/master/packages/jest-emotion) actually works with `shallow`.  
`jest-enzyme` (which i added) does have support for that however. Adding that package on the other hand breaks the shallow snapshot. Reason unknown. I assume it is due to the fact that it requires Theme?
So I use mount here instead.